File: pset1/greedy.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

float change;
int counter;

int main(void)
{
    do
    {
        printf("O hai! How much change is owed? ");
        change = round(GetFloat() * 100);
        counter = 0;
    } while (change <= 0);
    
    
    do
    {
        if (change >= 25)
        {
            change -= 25;
            counter++;
        } else if (change >= 10)
        {
            change -= 10;
            counter++;
        } else if (change >= 5)
        {
            change -= 5;
            counter++;
        } else if (change >= 1)
        {
            change -= 1;
            counter++;
        }
    }
    
    while (change > 0);
    
    printf("%i\n", counter);
}

File: pset6/greedy.py
amount = 0
goodinput = False
counter = 0
while not goodinput:
    try:
        number = float(input('How much is owed: '))
        if number > 0:
            goodinput = True
            amount = number
        else:
            print("Positive nunmber please!")
    except ValueError:
        print("That's not positive. Try again.")
        
print(amount * 100)

change = amount * 100


while change > 0:
    if change >= 25:
        change -= 25
        counter += 1
    elif change >= 10:
        change -= 10
        counter += 1
    elif change >= 5:
        change -= 5
        counter += 1
    elif change >= 1:
        change -= 1
        counter += 1
        
print(counter)